ui/page/accounts/accounts_page.go

```diff
@@ -116,7 +116,7 @@ func (pg *Page) fetchExchangeRate() {
 		return
 	}
 
-	rate := pg.AssetsManager.RateSource.GetTicker(market)
+	rate := pg.AssetsManager.RateSource.GetTicker(market, false) // okay to fetch latest rate, this is a goroutine
```

Comment:
this should have been true yeah? @itswisdomagain

Reply:
Nah, this method is called in a goroutine and network calls won't hang forever.